src/styles/components/code/styles.less

```diff
@@ -65,6 +75,11 @@
           right: @code-block-padding-right-screen-large;
           top: calc(@code-block-padding-top-screen-large ~' - 3px');
         }
+
+        .code-copy-icon.tight {
+          right: 0;
```

Comment:
can't we re-use theses properties?

Reply:
Re-use, meaning create a new generic class .right-0 or something for this?